modules/gear/loadouts/sf_medic.sqf

```diff
@@ -0,0 +1,29 @@
+
+/* Common Special Forces Gear */
+#include "sf_common.sqf"
+
+/* Primary Magazine Slots */
+ADD_MAGAZINE("30Rnd_556x45_Stanag", 7);
+/*ADD_MAGAZINE("30Rnd_556x45_StanagSD", 7);*/
+ADD_MAGAZINE("ACE_30Rnd_556x45_T_Stanag", 1);
+
+/* ACE Ruck Magazine Slots */
+ADD_MAGAZINE_RUCK("SmokeShellGreen", 2);
```

Comment:
Standard loadouts have medics having 2 hand grenades. If you want to add for standardisation its up to you

Reply:
I'm adding grenades and smoke through sf_common.sqf. Anything that they have in their individual gear assignment is extra.